lib/http.go

```diff
@@ -0,0 +1,50 @@
+package lib
+
+// Based on https://gist.github.com/josue/598ac048f78362f593f49436680b60b7
+
+import (
+	"log"
+	"net/http"
+	"strings"
+)
+
+/*
+GetURLHeaders returns a map array of all available headers.
+@param string - URL given
+@return map[string]interface{}
+*/
+func GetURLHeaders(url string) map[string]interface{} {
```

Comment:
Interested in why we are returning `map[string]interface` rather than `map[string]string` here,  since we know HTTP headers are going to be strings and we are casting to strings before assignment.

Reply:
Good point :)